packages/lesswrong/components/users/UsersMenu.tsx

```diff
@@ -96,148 +87,170 @@ const UsersMenu = ({classes}: {
-                <ListItemIcon>
-                  <BookmarksIcon className={classes.icon}/>
-                </ListItemIcon>
-                {isEAForum ? "Saved posts" : "Bookmarks"}
-              </MenuItem>
-            </Link>}
+              />
+            }
+            {!isEAForum &&
+              <DropdownItem
+                title={preferredHeadingCase("My Drafts")}
+                to="/drafts"
+                icon="Edit"
+                iconClassName={classes.icon}
+              />
+            }
+            {!currentUser.deleted &&
+              <DropdownItem
+                title={preferredHeadingCase("User Profile")}
+                to={`/users/${currentUser.slug}`}
+                icon="User"
+                iconClassName={classes.icon}
+              />
+            }
+            {userHasThemePicker(currentUser) &&
+              <ThemePickerMenu>
+                <DropdownItem
+                  title="Theme"
+                  onClick={() => {}}
+                  icon="Puzzle"
+                  iconClassName={classes.icon}
+                />
+              </ThemePickerMenu>
+            }
+            <DropdownItem
+              title={preferredHeadingCase("Account Settings")}
+              to="/account"
+              icon="Settings"
+              iconClassName={classes.icon}
+            />
+            <DropdownItem
+              title={preferredHeadingCase("Private Messages")}
+              to="/inbox"
+              icon="Email"
+              iconClassName={classes.icon}
+            />
+            {currentUser.bookmarkedPostsMetadata?.length > 0 &&
+              <DropdownItem
+                title={isEAForum ? "Saved posts" : "Bookmarks"}
+                to={isEAForum ? "/saved" : "/bookmarks"}
+                icon="Bookmarks"
+                iconClassName={classes.icon}
+              />
+            }
             {currentUser.shortformFeedId &&
-              <Link to={postGetPageUrl({_id:currentUser.shortformFeedId, slug: "shortform"})}>
-                <MenuItem>
-                  <ListItemIcon>
-                    <NotesIcon className={classes.icon} />
-                  </ListItemIcon>
-                  {preferredHeadingCase("Shortform Page")}
-                </MenuItem>
-              </Link>
+              <DropdownItem
+                title={preferredHeadingCase("Shortform Page")}
+                to={postGetPageUrl({
+                  _id: currentUser.shortformFeedId,
+                  slug: "shortform",
+                })}
+                icon="Shortform"
+                iconClassName={classes.icon}
+              />
             }
+
             <Divider/>
-            <a href="/logout">
-              <MenuItem>
-                {preferredHeadingCase("Log Out")}
-              </MenuItem>
-            </a>
-          </Paper>
-        </LWPopper>
+
+            <DropdownItem
+              title={preferredHeadingCase("Log Out")}
+              to="/logout"
```

Comment:
The logout button isn't working. I think this is a react router vs direct anchor issue, if I reload the logout page it does log me out

Reply:
Very good catch!